keyboards/smithrune/iron160/iron160_s/keymaps/default/keymap.c

```diff
@@ -31,5 +31,19 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
         _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
         _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______
-    )
+    ),
+    [2] = LAYOUT(
```

Comment:
if empty, not needed, and just adds to overall firmware size.

Reply:
Pushed in [1e6a6a6](https://github.com/qmk/qmk_firmware/pull/20616/commits/1e6a6a6f4f256e8353a5ea4abc381edfb9fb1c76)